python/setup.cfg

```diff
@@ -28,26 +28,4 @@ install_requires = cffi>=1.0.0
 opentracing = opentracing; jaeger_client
 
 [flake8]
-# TODO(PROTON-2322) decrease the limit
-max-line-length = 150
```

Comment:
I think I would've kept larger limit. Maybe not 150, but something like 120 sounds ok nowadays.

Reply:
Nvm, that's done below